Modules/_sqlite/connection.c

```diff
@@ -225,28 +225,50 @@ pysqlite_do_all_statements(pysqlite_Connection *self, int action,
     }
 }
 
+static int
+connection_traverse(pysqlite_Connection *self, visitproc visit, void *arg)
+{
+    Py_VISIT(self->statement_cache);
```

Comment:
`self->isolation_level` should be visited?

Reply:
> `self->isolation_level` should be visited?

I was under the impression that one does not need to visit `str` because a ref. cycle is impossible, no?

Reply:
OK. I have no experience in `sqlite` :(. But I believe that the `tp_traverse` have no side effect at least :)

Reply:
I've added it for consistency with the clear method :)

Reply:
Got it. Thanks.